### DISCUSSION TITLE
Example Sync Driver

### DISCUSSION THREAD

**Discussion by benutti** [sde1000/python-dali#133] [Q&A]
I'm trying to create something with the SyncTridonic driver, though I can't seem to find how to close the device. If I try to do dev.backend.close() , this doesn't work. 
I couldn't find an example of usage neither.

**Comment by rnixx** [comment]
Hi, basic usage example is here https://github.com/sde1000/python-dali/blob/master/dali/driver/tridonic.py#L307
Can you try this and get back with more information about possible errors if this not works?

**Reply by benutti** [reply]
I'm sorry, I'm quite new to python. You reffering me to a line in the driver for _test_sync doesn't help me that much.
I was able to close the device though doing dev.backend._device.reset() and then device.backend.close().
